nservicebus/operations/metrics-service-control.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Metrics data to ServiceControl
+summary: Reporting metrics to ServiceControl.
+reviewed: 2017-09-26
+component: MetricsServiceControl
+related:
+ - nservicebus/operations
+related:
+ - samples/logging/metrics
+---
+
+
+This component enables sending monitoring data gathered with NServiceBus.Metrics to an instance of ServiceControl.Monitoring service.
+
+## Enabling
+
+It can be enabled via:
+
+snippet: SendMetricDataToServiceControl
+
+
+#### Service Control Metrics Address
+
+The default is "particular.monitoring" which is ServiceControl monitoring instance input queue.
```

Comment:
Here also we can use ticks instead of quotes, I think it'll look better in terms of formatting so " => `